src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -12,37 +12,58 @@ module TaskSeq =
     /// <summary>
     /// Creates a <see cref="taskSeq" /> sequence from <paramref name="source" /> that generates a single element and then ends.
     /// </summary>
-    val singleton: source: 'T -> taskSeq<'T>
+    ///
+    /// <param name="value">The input item to use as the single value for the task sequence.</param>
+    val singleton: value: 'T -> taskSeq<'T>
 
     /// <summary>
     /// Returns <see cref="true" /> if the task sequence contains no elements, <see cref="false" /> otherwise.
     /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val isEmpty: source: taskSeq<'T> -> Task<bool>
 
     /// <summary>
     /// Returns the length of the sequence. This operation requires the whole sequence to be evaluated and
     /// should not be used on potentially infinite sequences, see <see cref="lengthOrMax" /> for an alternative.
     /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val length: source: taskSeq<'T> -> Task<int>
 
     /// <summary>
     /// Returns the length of the sequence, or <paramref name="max" />, whichever comes first. This operation requires the task sequence
     /// to be evaluated in full, or until <paramref name="max" /> items have been processed. Use this method instead of
     /// <see cref="TaskSeq.length" /> if you want to prevent too many items to be evaluated, or if the sequence is potentially infinite.
     /// </summary>
+    ///
+    /// <param name="max">The maximum value to return and the maximum items to count.</param>
+    /// <param name="source">The input task sequence.</param>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val lengthOrMax: max: int -> source: taskSeq<'T> -> Task<int>
 
     /// <summary>
     /// Returns the length of the sequence of all items for which the <paramref name="predicate" /> returns true.
     /// This operation requires the whole sequence to be evaluated and should not be used on potentially infinite sequences.
+    /// If <paramref name="predicate" /> is asynchronous, consider using <see cref="TaskSeq.lengthByAsync" />.
     /// </summary>
+    ///
+    /// <param name="predicate">A function to test whether an item in the input sequence should be included in the count.</param>
```

Comment:
Hm, I would have expect a `by` function's function param to be allowed to yield an effectiveLength.
Hm, maybe that's sumBy.... (just givign an instant reaction - not saying its wrong as such, esp if there is prior art)

Reply:
Leaving as is, function has been there for a while now, this is just documentation.